documentation/2.0/content/userguide/tools-config/logging.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Logging"
+date: 2022-02-04T17:18:00-05:00
+draft: false
+weight: 4
+---
+
+The WebLogic Deploy Tooling bas a built-in logging framework based on `java.util.logging`.  Its logging configuration
+is specified in `$WDT_HOME/etc/logging.properties`.  By default, the logging framework writes to both the console and
+a log file.
+
+#### Log File
+By default, WDT tools write their log files to the `$WDT_HOME/logs` directory and the log file name reflects the name of the tool.
+For example, if you run the `validateModel` tool then the log file will be `$WDT_HOME/logs/validateModel.log`.  These
+log files are overwritten each time you run a particular tool so the file only contains the logs from the last tool
+invocation.
+
+If the `$WDT_HOME/logs` directory is not writable by the user running the tool, the logging framework will search for
+a location to write the logs.  The user must have write permission on the directory in order for it to be selected.
+The search order is as follows:
+
+- Check the `WLSDEPLOY_LOG_DIRECTORY` environment variable.
+- Check the current working directory (as defined by the `user.dir` Java system property) and create a `logs` subdirectory.
+- Check the temp directory (as defined by the `java.io.tmpdir` Java system property) and create a `wdt-logs` subdirectory.
+
+If none of these locations are writable, the logging framework prints an error message to `stderr` and exits.
+
+#### Console Output
+WDT tools output logging information to `stdout` and `stderr`, as appropriate.  By default, only `INFO` level messages
+are sent to `stdout`.  All `WARNING` and `SEVERE` messages are set to `stderr`.  In addition to normal log messages
+generated as the tool runs, the tools will produce a summary at the end of tool execution that gives the user an
+overview of the tool execution status.  For example, the `validateModel` tool execution with no warnings or errors will
+produce output that looks similar to this:
+
+```
+Issue Log for validateModel version 2.0.0 running WebLogic version 12.2.1.4.0.210930 offline mode:
+
+Total:       WARNING :     0    SEVERE :     0
+```
+
+#### Logging Levels
+As mentioned previously, WDT's logging framework is based on `java.util.logging` so all logging levels defined in
+the `java.utiul.logging.Level` class apply to WDT loggers.  For a quick review of those levels, see the
+[javadoc](https://docs.oracle.com/javase/8/docs/api/java/util/logging/Level.html).
+
+WDT uses hierarchical loggers that align with what parts of the code is executing.  The root logger is named `wlsdeploy`.
```

Comment:
align with what parts of the code is executing -> (reads awkwardly) -> align with the executing parts of the code.